### PR TITLE
Read.me-format

### PR DESCRIPTION
**Pull request by rzenabi3**
edited read.me format ready for image of applications , references, and project status

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,52 @@
...
+## Description-Of-Code
+
+* Create quick search tool based off Google's Places API that allows users to quickly find medical professionals and institutions in a set area for set medical fields or types of service. We will provide users with a limited set of input values to ensure search options are limited to those in the medical field and services provided. Using Google Places API, we will start with the main function and focus being on finding basic information such as Name of professional and/or office, location, and contact info. As we get further in and if we feel the need, we will flesh out returned info more.
 We plan to supplement this with another API from a group like Yelp where we can let users see reviews or additional information from another source about the location they are inquiring about.
 
+
+## Installation
+
+* The code for this project is accessable through the git hub repository listed below in the REFERENCES section of this README.md.
+* Navigate to the repositry by directing into the Custom-Timed-Quiz repository located under the user "https://github.com/NewguyenGitHub"
```
Comment (NewguyenGitHub) [inline README.md:32]: Replace "Custom-Timed-Quiz" with "Recovery Radar"

File: README.md [hunk 2]
```diff
@@ -1,5 +1,52 @@
...
+* Once inside the Custom-Timed-Quiz repository click the green "<> Code" button and copy the HTTPS link.
```
Comment (NewguyenGitHub) [inline README.md]: Replace "Custom-Timed-Quiz" with "Recovery Radar"

File: README.md (hunk 1 above)
Comment (NewguyenGitHub) [inline README.md:32]: Replace "Custom-Timed-Quiz" with "Recovery Radar"

**Review by NewguyenGitHub — CHANGES_REQUESTED** [review]
On line 32 & 33, should it be replace "Custom-Timed-Quiz" with "Recovery Radar"?

**Comment by rzenabi3** [comment]
Wasn’t sure how to delete the previous pull request

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Pull request closed without merging**